base/stream.jl

```diff
@@ -844,6 +854,31 @@ function uv_write(s::LibuvStream, p::Ptr{UInt8}, n::UInt)
     return Int(n)
 end
 
+function uv_write_async(s::LibuvStream, p::Ptr{UInt8}, n::UInt)
```

Comment:
I'd like a comment here to explain what this function does (returns the last issued request?).